common/lib/dependabot/errors.rb

```diff
@@ -71,6 +71,15 @@ def initialize(source, msg = nil)
     end
   end
 
+  class PullRequestUpdateFailed < DependabotError
```

Comment:
Sorry for the slow review. Do we want to add this as a top-level error? Not sure we want to backport this to the other updaters, for github there are so many different reasons for a PR update failing so will be hard to reuse there. Maybe better to just add it in the Azure class for now?

Reply:
Cool makes sense! I'll add this in azure pr updater class